dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_12__Fix_id_column_type.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE programruleaction ALTER COLUMN notificationtemplateid TYPE BIGINT;
```

Comment:
new jpa schema validator complain about this column has type `integer` in database but `long` in Java